exercises/practice/diamond/diamond_test.go

```diff
@@ -103,7 +103,7 @@ func TestDiamondIsHorizontallySymmetric(t *testing.T) {
 func TestDiamondIsVerticallySymmetric(t *testing.T) {
 	requirement := func(char byte, rows []string) bool {
 		for i, j := 0, len(rows)-1; i < j; i, j = i+1, j-1 {
-			if rows[i] != rows[j] {
+			if strings.TrimSpace(rows[i]) != strings.TrimSpace(rows[j]) {
```

Comment:
Why is this needed?

Reply:
Not needed anymore as we reverted the `\n` at the end